content/blog/2022/dependabot-single-pull-request.md

```diff
@@ -47,11 +49,11 @@ to upgrade dependencies at our convenience.
 Broadly speaking, the workflow will:
 
 1. Checkout our code
-1. Configure the runtimes required by our project (Python and Node)
-1. Run a custom command to upgrade all dependencies
-1. Use git to detect changed files
-1. Commit the changes to a pre-defined branch (if they exist)
-1. Open a pull request for that branch (if it doesn't exist already)
+2. Configure the runtimes required by our project (Python and Node)
+3. Run a custom command to upgrade all dependencies
+4. Use git to detect changed files
+5. Commit the changes to a pre-defined branch (if they exist)
+6. Open a pull request for that branch (if it doesn't exist already)
```

Comment:
Was this change on purpose? This was an intentional use of Markdown's auto-numbering: https://daringfireball.net/projects/markdown/syntax#list

Reply:
nope, this must have been a linter?